oss-cloud-backend/handler.js

```diff
@@ -113,23 +110,42 @@ module.exports.addContributor = async (event, context, callback) => {
 // {
 //    username: string
 // }
-module.exports.updatePullRequests = async (event, context, callback) => {
+module.exports.updatePullRequests = async () => {
   let response;
   try {
     const results = await gitHubApiService.updatePullRequests();
     response = {
       statusCode: 200,
-      body: results.toString() + " contributors updated"
+      body: `${results.toString()} contributors updated`,
```

Comment:
You don't need toString() here, it will be converted to string anyway.

Reply:
Removed toString()